web-ui/src/context/selectors.js

```diff
@@ -16,9 +16,20 @@ export const selectLoading = state => state.loading;
 export const selectReviewPeriods = state => state.reviewPeriods;
 export const selectPermissions = state => state.permissions;
 
+const hasPermission = permissionName => createSelector(
```

Comment:
I removed a lot of code duplication by defining this function.

Reply:
So much better to read!